mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/stream/TopPanelRow.java

```diff
@@ -91,6 +100,13 @@ public void bind(Cursor cursor, int tilesSize) {
         swipeListener.currentPosition = 0;
     }
 
+    /* Cliqz Start */
+    public void setLightTheme() {
```

Comment:
You are setting nothing, the name is misleading. A better one will be `updateTheme`.